public/app/features/dashboard/components/PanelEditor/PanelEditor.tsx

```diff
@@ -149,6 +149,16 @@ export class PanelEditorUnconnected extends PureComponent<Props> {
   };
 
   onFieldConfigChange = (config: FieldConfigSource) => {
+    if (this.props.panel.libraryPanel?.uid !== undefined) {
+      this.props.dashboard.panels.forEach((panel) => {
+        if (panel.libraryPanel?.uid === this.props.panel.libraryPanel!.uid) {
+          panel.updateFieldConfig({
+            ...config,
+          });
+        }
+      });
+    }
```

Comment:
not sure I understand this condition and what is going on here

Reply:
the panel edit cannot update other panels in the dashboard (would break discard mechanic)